crates/ide/src/inlay_hints/implicit_drop.rs

```diff
@@ -1,10 +1,8 @@
 //! Implementation of "implicit drop" inlay hints:
-//! ```no_run
-//! fn main() {
-//!     let x = vec![2];
-//!     if some_condition() {
-//!         /* drop(x) */return;
-//!     }
+//! ```compile_fail
```

Comment:
We could use `ignore` so we don't try to compile this.

Reply:
addressed:

- https://github.com/rust-lang/rust-analyzer/pull/16519/commits/a3e60e7f7af383d7b067c12cde78c02d8c15ee26
- https://github.com/rust-lang/rust-analyzer/pull/16519/commits/adddd14afb63b1646b4ac5eb4d33a611c9c864d5